src/resource-extensions/resource-extensions.js

```diff
@@ -64,7 +64,7 @@
         var DEFAULT_ACTIONS = {
           'get': { method: 'GET' },
           'save': { method: 'POST' },
-          'query': { method: 'GET', isArray: true },
+          'query': { method: 'GET', isArray: true, cancellable: true },

```

Comment:
Has it been investigated whether we can write a unit test for this? I'd like to maintain high coverage